mne/viz/tests/test_evoked.py

```diff
@@ -30,6 +30,7 @@
 
 warnings.simplefilter('always')  # enable b/c these tests throw warnings
 
+rng = np.random.RandomState(0)
```

Comment:
better to nest this in the function that uses it, lest the next contributor add a function and think it's okay to use the same `rng` (it could break your test with a heisenbug)